virtac/atip_ioc_entry.py

```diff
@@ -21,14 +23,16 @@ def parse_arguments():
     parser.add_argument(
         "--disable-emittance", "-d", help="disable emittance calc", action="store_true"
     )
+    parser.add_argument(
+        "--enable-tfb", "-t", help="enable dummy Tune Feedback", action="store_true"
```

Comment:
Should this be 'enable dummy Tune Feedback PVs'?

Reply:
Is this dummy PVs pretending to be the TFB system, or PVs pretending to be the underlying hardware required for TFB to operate?

It's still not clear from the description

Reply:
The underlying hardware required for TFB to operate

Reply:
I've gone with "enable dummy hardware to be used by Tune Feedback", hopefully that's clearer